cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,30 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
+  let user;
   before(() => {
```

Comment:
you do not need this before hook. You can log in in the first test and create a const with user in the describe block